src/BedrockStreaming.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace M6Web\CS\Config;
+
+use PhpCsFixer\Config;
+
+final class BedrockStreaming extends Config
+{
+    public function __construct()
+    {
+        parent::__construct('Bedrock Streaming');
+
+        $this->setRiskyAllowed(true);
+    }
+
+    public function getRules(): array
+    {
+        $rules = [
+            '@Symfony' => true,
+            'array_syntax' => [
+                'syntax' => 'short',
+            ],
+            'braces' => [
+                'allow_single_line_closure' => true,
+            ],
+            'global_namespace_import' => [
+                'import_classes' => false,
+                'import_constants' => false,
+                'import_functions' => false,
+            ],
+            'heredoc_to_nowdoc' => false,
+            'increment_style' => ['style' => 'post'],
+            'no_unreachable_default_argument_value' => false,
+            'ordered_imports' => ['sort_algorithm' => 'alpha'],
+            'phpdoc_line_span' => [
+                'property' => 'single',
+                'const' => 'single',
+            ],
+            'phpdoc_summary' => false,
+            'single_line_throw' => false,
+            'yoda_style' => false,
```

Comment:
`'declare_strict_types' => true`
?

Reply:
I have my doubt about this one, but I will trust you (two 😉)

Reply:
yes, this is a tough one... I'd love to have this rule applied on all our new files... but the problem is that it will be applied to old files too... so I'm not so sure that this is a good idea now 😅 

reference: https://mlocati.github.io/php-cs-fixer-configurator/#version:3.0|fixer:declare_strict_types

Reply:
Ok, removing it for now 😅